templates/cluster_resources.j2

```diff
@@ -98,7 +114,9 @@ primitive rsc_sap_{{ sid }}_ASCS{{ ascs_instance }} SAPInstance \
      migration-threshold=1 priority=10
 
 group grp_{{ sid }}_ASCS{{ ascs_instance }} \
-  rsc_ip_{{ sid }}_ASCS{{ ascs_instance }} rsc_fs_{{ sid }}_ASCS{{ ascs_instance }} rsc_sap_{{ sid }}_ASCS{{ ascs_instance }} {% if cloud_provider == "microsoft-azure" %} nc_{{ sid }}_ASCS {% endif %} \
+  rsc_ip_{{ sid }}_ASCS{{ ascs_instance }} rsc_fs_{{ sid }}_ASCS{{ ascs_instance }} rsc_sap_{{ sid }}_ASCS{{ ascs_instance }}
+  {%- if cloud_provider == "microsoft-azure" %} nc_{{ sid }}_ASCS {% endif -%}
+  {%- if data.monitoring_enabled %} rsc_exporter_{{ sid }}_ASCS{{ ascs_instance }} {% endif -%}
```

Comment:
I increased the `resource-stickiness` to `4000` in my tests and works fine. Need to really understand what we need to put here

Reply:
I don't quite understand, what makes 4000 better than the old value 3000?

Reply:
To be honest, no idea... I remember some days ago that @diegoakechi told us something about this. As far as the migration is done, I don't really see any difference.

Info:
- https://clusterlabs.org/pacemaker/doc/en-US/Pacemaker/1.1/html/Pacemaker_Explained/s-resource-options.html
- https://clusterlabs.org/pacemaker/doc/en-US/Pacemaker/1.1/html/Clusters_from_Scratch/_prevent_resources_from_moving_after_recovery.html

Reply:
@stefanotorresi @arbulu89 It should be ok to keep 3000, as the group receives 3000 not the resource. The problem I had was on the yast module, since it configures the stickiness to 1000, so every new resource sum more 1000 on the stickiness calculation, but it got improved on the current best practices.

Reply:
so should I leave it to 3000 then?

Reply:
On my POV, yes, but it is more important to test it and see if works. Basically the resources on the group needs to migrate together with the ASCS instance in case of failover. If it works, to 3000 is fine.

Reply:
okay, thanks for your input Diego! It appears to work quite well, I will test this further.